EIPS/eip-2537.md

```diff
@@ -67,10 +69,10 @@ Twist type: M
 B coefficient for twist c0 = 0x000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000004
 B coefficient for twist c1 = 0x000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000004
 Generators:
-G1:
+H1:
```

Comment:
why H and not G?

Reply:
Just to resolve ambiguity. Because G1 is used for a group of points.